src/hera.cpp

```diff
@@ -154,8 +154,12 @@ vector<uint8_t> evm2wasm_js(vector<uint8_t> const& input) {
   os.open(fileEVM);
   // print as a hex sting
   os << hex;
-  for (uint8_t byte: input)
-    os << static_cast<int>(byte);
+  for (uint8_t byte: input) {
+    int val = static_cast<int>(byte);
+    if (val<10)
+      os << 0;
```

Comment:
Please use something like this (from https://stackoverflow.com/questions/5760252/how-can-i-print-0x0a-instead-of-0xa-using-cout):
`setfill('0') << setw(2) << static_cast<int>(byte)`